taichi/backends/device.h

```diff
@@ -400,7 +400,7 @@ class Device {
   void print_all_cap() const;
 
   struct AllocParams {
-    uint64_t size{0};
+    size_t size{0};
```

Comment:
Why do we need this change?

Reply:
Mostly code cleanup: I think size_t is the idiomatic way to represent size. Do you have specific concerns here? (One thing I can think of is that, size_t is platform-dependent)

Reply:
Yes, shouldn't use platform dependent types here. It can be sometimes dangerous if these types are used (implicit casting, etc.)